### ISSUE TITLE
thread 'actix-rt|system:0|arbiter:1' has overflowed its stack

### ISSUE THREAD

**Issue by scholtz** [near/nearup#218]
nearup works fine until it crashes with error:

```
thread 'actix-rt|system:0|arbiter:1' has overflowed its stack
fatal runtime error: stack overflow
```

![image](https://user-images.githubusercontent.com/1223439/176865547-64ea69c4-e2f1-4417-9345-75b337372a2a.png)

**Comment by DDeAlmeida** [comment]
Related to nearcore not nearup.
Can be closed @mina86 @nikurt

**Comment by mina86** [comment]
This should be fixed with 1.28.0-rc.3.  Please reopen if this keeps happening after node update.
